icons/vans.svg

```diff
@@ -0,0 +1,2 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 24 24" viewBox="0 0 24 24"><path fill="#F44336" d="M8.69,17H7l1.891-6.702h1.748L12.53,17h-1.719l-0.201-0.874H8.891L8.69,17z M10.324,14.705l-0.573-2.404l-0.544,2.404H10.324z"/><polygon fill="#F44336" points="14.951 10.298 16.842 13.941 16.785 12.411 16.785 10.298 18.447 10.298 18.447 17 16.814 17 14.922 13.358 15.008 14.888 15.008 17 13.346 17 13.346 10.298"/><path fill="#F44336" d="M23.857,12.324h-1.633c-0.019-0.209-0.105-0.349-0.258-0.419c-0.096-0.093-0.239-0.14-0.43-0.14c-0.382,0-0.573,0.128-0.573,0.384c0,0.233,0.344,0.454,1.032,0.663C23.331,13.208,24,13.883,24,14.836c-0.039,1.419-0.86,2.14-2.465,2.164c-1.395-0.07-2.178-0.768-2.35-2.094h1.662c0.057,0.233,0.153,0.407,0.286,0.523c0.134,0.093,0.296,0.14,0.487,0.14c0.497,0,0.745-0.151,0.745-0.454c0-0.256-0.334-0.477-1.003-0.663c-1.357-0.395-2.035-1.093-2.035-2.094c0.038-1.326,0.793-2.012,2.264-2.059C22.93,10.323,23.685,10.998,23.857,12.324L23.857,12.324z"/><rect width="17.368" height="1.5" x="6.714" y="7" fill="#F44336"/><polygon fill="#F44336" points="4.336 14.692 1.891 7.016 0 7.009 3.127 17 5.433 17 8.498 7 6.627 7"/></svg>
```

Comment:
you should add `title` like this `<title>Adobe Photoshop</title>`, what's more, i think `enable-background="new 0 0 24 24"` should be removed. Here is a example:
`<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Adobe Photoshop</title>...</svg>`🧐

Reply:
You are correct @kerthcet, a complete overview of what should and shouldn't be here can be found in [section 4 of the Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#4-annotate-the-icon)